src/components/user/avatar/UserAvatar.tsx

```diff
@@ -51,7 +48,7 @@ export default class UserAvatar extends React.Component<Props, State> {
   };
 
   public render() {
-    const { selected, small } = this.props;
+    const { accessoryIcon, small } = this.props;
```

Comment:
The checked icon should not be part of the avatar.
The approach of selecting an element in the list should be global even if there is no Avatar (Sites, Site Areas...)
So for single selection, I would like a checkbox and for multi a radio button and they should be on the left of each item.

Reply:
The checked icon is not part of the Avatar, I just offer the possibility to decorate the avatar with an accessory of choice, and use it to indicate that a user is selected in the user list.

Reply:
Okay but not okay ;-)

Reply:
Do I completely remove the accessory ?

Reply:
Should IOS and Android have the exact same components for checkbox and radio buttons or is it okay let the lib use native components ?. Because IOS have the same radio button and checkbox, with nothing rendered until selected, thus leaving a blank space as shown on the pic 

<img width="1017" alt="Screenshot 2021-04-24 at 04 14 22" src="https://user-images.githubusercontent.com/32574035/115944045-dd125700-a4b3-11eb-9f76-115b5c5a4d70.png">

Reply:
You have to make sure the components choice by OS only impact TS code. Otherwise it's gonna be a maintenance nightmare to have react-native wrappers for components.

Reply:
My question is precisely, should I code the components myself, independently of the OS or should I let the lib use native and then different components for each OS

Reply:
Questions to ask yourself before introducing code covering already existing features with code:
+ Why do I need it or what the code is meant to solve? 
+ What will be its maintenance overhead? 
+ Is there a well maintained FOSS third-party doing it the desired way?

Reply:
Solved, I managed to have the same component with a lib we already use